all/guides/wix-groups/docs/Introduction.md

```diff
@@ -1,24 +1,29 @@
+With the Groups APIs, you can:
+- Create and manage [groups](https://dev.wix.com/docs/rest/crm/community/groups/groups/introduction).
+- Manage group [rules](https://dev.wix.com/docs/rest/crm/community/groups/rules/introduction).
 - Manage [requests](https://dev.wix.com/docs/rest/crm/community/groups/create-requests/introduction) to create new groups.
+- Manage group [members](https://dev.wix.com/docs/rest/crm/community/groups/members/introduction).
+- Manage group member [roles](https://dev.wix.com/docs/rest/crm/community/groups/roles/introduction).
 - Manage member [requests to join](https://dev.wix.com/docs/rest/crm/community/groups/join-requests/introduction) groups.
-- Manage a group's [rules](https://dev.wix.com/docs/rest/crm/community/groups/rules/introduction) and [questions](https://dev.wix.com/docs/rest/crm/community/groups/membership-questions/introduction) to join.
+- Manage [membership questions](https://dev.wix.com/docs/rest/crm/community/groups/membership-questions/introduction) for prospective members. 
 
-Read more about [Wix Groups](https://support.wix.com/en/article/wix-groups-about-groups).
+Read more about [Wix Groups](https://support.wix.com/en/article/wix-groups-about-groups). 
 
 ## Use cases
-- [Create a group for a predefined list of people](https://dev.wix.com/docs/api-reference/crm/community/groups/example-flow#create-a-group-for-a-predefined-list-of-people).
-- [Manage bulk emailing to group members](https://dev.wix.com/docs/api-reference/crm/community/groups/example-flow#manage-bulk-emailing-to-group-members).
-- [Provide analytical reports](https://dev.wix.com/docs/api-reference/crm/community/groups/example-flow#provide-analytical-reports).
-- [Send promotional emails based on members interests](https://dev.wix.com/docs/api-reference/crm/community/groups/example-flow#send-promotional-emails-based-on-members-interests).
-- [Bulk update group rules](https://dev.wix.com/docs/api-reference/crm/community/groups/example-flow#bulk-update-group-rules).
+
+This article presents possible use cases and corresponding sample flows that you can support. This can be a helpful jumping off point as you plan your implementation.
+
+- [Create a group for a predefined list of people](https://dev.wix.com/docs/rest/crm/community/groups/example-flow#create-a-group-for-a-predefined-list-of-people).
+- [Manage bulk emailing to group members](https://dev.wix.com/docs/rest/crm/community/groups/example-flow#manage-bulk-emailing-to-group-members).
+- [Provide analytical reports](https://dev.wix.com/docs/rest/crm/community/groups/example-flow#provide-analytical-reports).
+- [Send promotional emails based on members' interests](https://dev.wix.com/docs/rest/crm/community/groups/example-flow#send-promotional-emails-based-on-members-interests).
+- [Bulk update group rules](https://dev.wix.com/docs/rest/crm/community/groups/example-flow#bulk-update-group-rules).
```

Comment:
You should change the title (in the sidebar menu) of the Example Flow to Sample Flows. This will necessitate correcting the links here.